src/extensions/index.ts

```diff
@@ -128,6 +128,9 @@ export class ExtensionManager implements Extension {
     };
     extensions :Partial<Extension>[];
 
+        // holds the conflicts which are occurs during the timeline creation
+    conflicts: {event_body: Conflict<EventBody>[]} = {event_body: []};
```

Comment:
This exporting object is for all extensions. But the property you added is only for ConflictFixers, so you cannot add here